### DISCUSSION TITLE
any way to save frags on respawn?

### DISCUSSION THREAD

**Discussion by itsmahdiso** [SmileYzn/PugMod-AMXX#45] [Ideas]
I think if, like the rounds,  its good to  players frags (kill and death) are saved on team swaps and overtime too

Can this be added?

**Comment by SmileYzn** [comment]
Can be added, i need only to find a efficient way to do it, since is using re-api

**Reply by itsmahdiso** [reply]
Can you do that please?

**Reply by SmileYzn** [reply]
I need to project in a separated plugin first, since the pugMod do some restarts in game.
And in some situations need to reset scores and others not :D

I will check it
